RecoEgamma/EgammaPhotonProducers/python/photons_cfi.py

```diff
@@ -80,7 +80,13 @@
     RecHitSeverityToBeExcludedEE = cleanedHybridSuperClusters.RecHitSeverityToBeExcluded,
 )
 
-photonsFromMultiCl = photons.clone(
-  photonProducer = 'photonCoreFromMultiCl'
-)
-
+photonsFromMultiCl = photons.clone()
+# Change underlying C++ type, as ugly as it could be, and add required
+# configuration parameters
```

Comment:
I only want to note that parameters can be deleted when cloning/toModifying with
```
photonsFromMultiCl = _gedPhotons.clone(
    ...
    regressionConfig = None
)
```